service/templates/admin.html

```diff
@@ -1237,6 +1237,10 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 									<input type="number" ng-model="Settings.amboy.group_ttl">
 									</md-input-container>
 
+				          <md-input-container class="control" style="width:45%;">
+									<label>lock timeout</label>
```

Comment:
The label should be in sentence case

Reply:
```suggestion
									<label>Lock timeout</label>
```
Like that? also thanks for the guidance regarding the Ticket #

Reply:
Yep, exactly

Reply:
Can we have it read "Lock timeout (minutes)"